ci/windows-pkg-arrow-for-r.sh

```diff
@@ -28,39 +28,51 @@ mkdir $DST_DIR
 # Grab the headers from one, either one is fine
 mv mingw64/include $DST_DIR
 
-# Move the 64-bit versions into the expected location
+# Make the rest of the directory structure
+# lib-4.9.3 is for libraries compiled with gcc 4.9 (Rtools 3.5)
 mkdir -p $DST_DIR/lib-4.9.3/x64
+mkdir -p $DST_DIR/lib-4.9.3/i386
+# lib is for the new gcc 8 toolchain (Rtools 4.0)
```

Comment:
gcc 8, really? Does that mean we can ditch gcc 4.9 at some point?

Reply:
No :) We want to keep supporting that too.